### PR TITLE
CIV 1843 

### PR DESCRIPTION
**Pull request by suvarnathekkoot**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-1843

### Change description ###

Disposal Hearing Order details page - Static Fields

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/CaseEventToFields/CreateSDO-SDO-nonprod.json [hunk 1]
```diff
@@ -120,5 +120,192 @@
     "PageDisplayOrder": 4,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CREATE_SDO",
+    "CaseFieldID": "disposalOrderAndHearingDetails",
```
Comment (Gulam-Hussain) [inline ccd-definition/CaseEventToFields/CreateSDO-SDO-nonprod.json:127]: think authorisation for this is missing

File: ccd-definition/CaseEventToFields/CreateSDO-SDO-nonprod.json [hunk 2]
```diff
@@ -120,5 +120,192 @@
...
+    "DisplayContext": "COMPLEX",
+    "PageID": "DisposalHearing",
+    "PageDisplayOrder": 5,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CREATE_SDO",
+    "CaseFieldID": "disposalClaimSettling",
```
Comment (Gulam-Hussain) [inline ccd-definition/CaseEventToFields/CreateSDO-SDO-nonprod.json:270]: think authorisation for this is missing

File: ccd-definition/CaseEventToFields/CreateSDO-SDO-nonprod.json [hunk 3]
```diff
@@ -120,5 +120,192 @@
...
+    "DisplayContext": "READONLY",
+    "PageID": "DisposalHearing",
+    "PageDisplayOrder": 5,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CREATE_SDO",
+    "CaseFieldID": "disposalCosts",
```
Comment (Gulam-Hussain) [inline ccd-definition/CaseEventToFields/CreateSDO-SDO-nonprod.json:281]: think authorisation for this is missing

File: ccd-definition/CaseEventToFields/CreateSDO-SDO-nonprod.json [hunk 4]
```diff
@@ -120,5 +120,192 @@
...
+    "DisplayContext": "READONLY",
+    "PageID": "DisposalHearing",
+    "PageDisplayOrder": 5,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CREATE_SDO",
+    "CaseFieldID": "disposalApplicationsOrder",
```
Comment (Gulam-Hussain) [inline ccd-definition/CaseEventToFields/CreateSDO-SDO-nonprod.json:292]: think authorisation for this is missing

**Review by Gulam-Hussain — DISMISSED** [review]

**Review by Gulam-Hussain — APPROVED** [review]

**Review by cj1993 — APPROVED** [review]

**Pull request closed without merging**